src/main/scala/nl/knaw/dans/easy/stage/Settings.scala

```diff
@@ -39,9 +39,9 @@ case class Settings(ownerId: String,
 object Settings {
 
   /** backward compatible call for EasyIngestFlow */
-  def apply(ownerId: String,
+  def apply(
```

Comment:
Note that this change implies you have to change EasyIngestFlow too, run `mvn clean-install` of `easy-build` to make sure all components are happy with the new API.